stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -593,13 +601,13 @@ public class MessageListView : ConstraintLayout {
                 String.format("Scroll: First load scrolling down to bottom %d", newPosition)
             )
             lastMessageReadHandler.invoke()
-        } else if (listItem.loadingMore) {
-            // the load more behaviour is different, scroll positions starts out at 0
-            // to stay at the relative 0 we should go to 0 + size of new messages...
-
-            // TODO ???
-            val newPosition = layoutManager.findLastCompletelyVisibleItemPosition() + sizeGrewBy
-            layoutManager.scrollToPosition(newPosition)
+//        } else if (listItem.loadingMore) {
```

Comment:
Don't we need it anymore?
On that case we should remove this code

Reply:
We never had the `listItem.loadingMore` as true before, so this never executed. When it started to execute, it was causing a scroll every time we load more data, which was not something to be expected. Let's remove it.